client/AddTask.jsx

```diff
@@ -46,7 +46,13 @@ class AddTask extends React.Component {
       return;
     }
 
-    this.props.onAddNewTask(taskName, dueDate);
+    socket.emit('create task', {
```

Comment:
Move this into a 'frontend' service.. which is the new feature you are working on now